gremlin-go/driver/strategies_test.go

```diff
@@ -47,4 +47,169 @@ func TestReadOnlyStrategy(t *testing.T) {
 		assert.Nil(t, err)
 		assert.NotNil(t, <-promise)
 	})
+
```

Comment:
Should these tests be simple count() queries, or should we try to make graphs and execute traversals that would display the optimized result after applying these strategies?

Reply:
there are no integration tests for python and .net driver, only basic tests for generating byte-code.
I think gremlin-core package is responsible for testing the implementation of the strategy.

Reply:
ACK